src/lib/pagination/tabs.tsx

```
@@ -56,43 +1,147 @@
+   */
+  tabPanelProps?: TabPanelProps;
 }
 
-interface TabsProps {
-  currentValue: any;
+interface TabsProps extends Omit<AriaTabsProps, "orientation"> {
   items: TabsItem[];
-  //eslint-disable-next-line @typescript-eslint/ban-types
-  callback: Function;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+  callback?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)